### ISSUE TITLE
JAWS/Chrome focus indicator not consistently visible on screen

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#38]
This is an issue in reference to #25 which is related to the ["region" branch](https://github.com/fizzstudio/svg-aria-test/tree/aria_region).

<img width="1063" alt="Screenshot of JAWS in test environment. A red circle is highlighting the Text Viewer text at the top of the screen that reads '3 Text' but the text is not visible- it is completely off screen." src="https://user-images.githubusercontent.com/98043076/153679081-26d1d429-0061-4708-b820-12e4577af773.png">

### Issue Summary
JAWS does not adjust the screen to show targeted text visually. A sighted/low vision user would need to manually scroll to view all of the SVG element clearly. The video provided was recorded without any manual scrolling or pre-setup on the page. 

### Video of issue 
Example Issue 1 starts at 00:00:06
https://user-images.githubusercontent.com/98043076/154173460-8261b974-a583-4936-8c71-6b8d9233af35.mp4

### Steps to reproduce
Use JAWS down arrow key once to navigate into the SVG space. Repeat this 4-6 times to get to the "path and text" group element.

### Expected behavior (Pass/Fail)
- **FAIL** We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- **Pass** We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- **FAIL** We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating
- **FAIL** The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- **FAIL** The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- **FAIL** The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- **Pass** Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- **Pass** We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- **Pass** The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed viusally in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- **FAIL** We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- **FAIL** We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).

### Technical Details
JAWS 2022.2112.24 ILM
Chrome Version 97.0.4692.99
Windows 11 Build 22000.438
